flow/Makefile

```diff
@@ -294,7 +294,7 @@ synth: versions.txt \
 #-------------------------------------------------------------------------------
 SYNTH_SCRIPT ?= $(SCRIPTS_DIR)/synth.tcl
 
-$(RESULTS_DIR)/1_1_yosys.v: $(DONT_USE_LIBS) $(WRAPPED_LIBS) $(DONT_USE_SC_LIB) $(VERILOG_FILES)
+$(RESULTS_DIR)/1_1_yosys.v: $(DONT_USE_LIBS) $(WRAPPED_LIBS) $(DONT_USE_SC_LIB) $(DFF_LIB_FILE) $(VERILOG_FILES) $(CACHED_NETLIST) $(LATCH_MAP_FILE) $(ADDER_MAP_FILE)
```

Comment:
Not that you put this here, but it doesn't make sense to me why there is both `$(DONT_USE_LIBS)` and `$(DONT_USE_SC_LIB)`... if I understand things correctly `$(DONT_USE_SC_LIB)` is just the first item in `$(DONT_USE_LIBS)`. I'll probably fix that after this PR is merged.

Reply:
You are correct, I can fix that while I'm in here anyways.

Reply:
@rovinski it looks like `$(DONT_USE_SC_LIB)` is defined as ?= so its possible this file is not the first from the DONT_USE_LIBS (although ill advised, it's possible the library is not part of the DONT_USE_LIBS variable), so maybe its best to leave it? If the version of Yosys is updated to the one that supports giving multiple lib files to abc, then the need for this variable is gone and would be replaced with the `DFF_LIB_FILE`.

Reply:
You're probably right. That variable was added for ASAP7 which IMO that whole platform is a mess, and I don't really want to touch it.

With respect to the Yosys update, I 100% agree, but the problem is that I tried this a little bit ago and the new Yosys version ended up blowing up some of our QoR metrics. So that update got put on hold until either we get our metrics in a better state or until Yosys provides better QoR. Another issue is going through all of our cached netlists and updating them to the new version.